### PR TITLE
Us10 s06 before home

### PR DESCRIPTION
**Pull request by sorrento974**

### CODE REVIEW

File: client/src/main.jsx [hunk 1]
```diff
@@ -31,11 +33,15 @@ import Admin from "./pages/Admin";
 import Score from "./pages/Score";
 
 const router = createBrowserRouter([
+  {
+    path: "/",
```
Comment (Dinosaure7) [inline client/src/main.jsx:39]: the path / is for the home page, you can use /BeforeHome for example for the path of the pre-homepage

**Review by Dinosaure7 — CHANGES_REQUESTED** [review]
Review the pre-hompage route

**Review by Alice-Berthelot — CHANGES_REQUESTED** [review]
images about street art should be stocked in server, not client (server > public > assets > images)

File: client/src/main.jsx (hunk 1 above)
Comment (Alice-Berthelot) [reply client/src/main.jsx:39]: yes the path for BeforeHome is "/", however this path "/" has been used until now for our Home page so you should carefully change all the links that point to "/" to "/home" , on all the pages of our application

also BeforeHome is a child of App, so it should be moved down

**Review by Alice-Berthelot — CHANGES_REQUESTED** [review]

**Review by Dinosaure7 — APPROVED** [review]

**Review by Yoann-Fortin — APPROVED** [review]

**Review by Yoann-Fortin — APPROVED** [review]

**Review by Yoann-Fortin — APPROVED** [review]

**Pull request merged**